pandas/io/formats/console.py

```diff
@@ -21,7 +21,7 @@ def detect_console_encoding():
     encoding = None
     try:
         encoding = sys.stdout.encoding or sys.stdin.encoding
-    except AttributeError:
+    except Exception:
```

Comment:
Why can't we just `except` both on `AttributeError` and `IOError` ?  I generally try to avoid using the broad `Exception` unless we have good reason to.